server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/MailboxSetMethodContract.scala

```diff
@@ -3027,7 +3027,6 @@ trait MailboxSetMethodContract {
          |}""".stripMargin)
   }
 
-  // TODO testing delegation & partial updates
   // TODO write test for rights store manager applyCommandById in MailboxManagerTest
```

Comment:
Can we remove the TODOs from the commit history?

Reply:
outdated?

Reply:
No I meant can we remove all traces of their existence?